src/favouritesPage/components/_favouriteCardMap.scss

```diff
@@ -0,0 +1,7 @@
+.FavouriteCardMap {
+  height: 100%;
+  &__icon {
+    color: $color-gold;
+    fill: currentColor;
```

Comment:
Sorry, I was a bit unclear last time; it'd specifically have to be `path { fill: currentColor }` nested here to work. However, I checked and the HDS icons are designed in a robust way by including this rule in their SVG paths already, so we actually don't need to declare that at all, just the color ones already do the trick.